vortexasdk/endpoints/voyages_search_enriched.py

```diff
@@ -294,3 +294,259 @@ def search(
         }
 
         return VoyagesSearchEnrichedResult(super().search(headers=self._HEADERS, **api_params))
+
+    def search_raw(
```

Comment:
`search_raw` doesn't send `text/csv` headers or `columns` API params to the server - the server returns only a JSON response.